database/connection.go

```diff
@@ -17,3 +24,20 @@ func NewPool(url string) (*pgxpool.Pool, *sqlc.Queries) {
 
 	return pool, queries
 }
+
+func RunMigrations(databaseURL string) {
+	u, err := url.Parse(databaseURL)
+	if err != nil {
+		log.Fatal().Err(err).Msg("Invalid database URL")
+	}
+
+	db := dbmate.New(u)
+	db.FS = fs // Set the embedded filesystem for migrations
+	db.MigrationsDir = []string{"migrations"}
+	db.AutoDumpSchema = false
```

Comment:
I will create a build CI step later to make sure that this does compile, this reminded me of that